docs/workloads/azure/data/getting_started/fabric_deployment_guide.md

```diff
@@ -0,0 +1,71 @@
+---
+id: Quick step by step guide for fabric deployment
+title: Quick step by step guide for fabric deployment
+sidebar_label: 2. Infrastructure Deployment
+hide_title: false
+hide_table_of_contents: false
+description: Deployment of common Azure Data Factory resources shared by data pipelines
+keywords:
+  - data
+  - infrastructure
+  - adf
+  - cicd
+  - fabric
+---
+
+This section provides an overview of deploying microsoft fabric through Azure portal for Ensono Stacks Data Platform.
+
+
+## Step 1: Create fabric capacity resource
+
+1. Login to azure portal, search for Fabric and select Create.
+
+![Ensono Stacks Fabric look](../images/fabric_lookup.png)
+
+2. Create a fabric resource.
+![Ensono Stacks Fabric create](../images/fabric_create.png)
+   1. Select the subscription for the new fabric capacity.
+   2. Create a new resource group for this fabric capacity, or select an existing one.
+   3. Enter the name used to identify this resource and is displayed in the Microsoft Fabric admin portal and Azure portal. The name must be unique in the selected location. Only lowercase letters and numbers may be used.
+   4. Enter the region.
+   5. Select the resource size that best meets your needs.
+   6. Microsoft Fabric capacity administrator will manage the capacity. The capacity administrator must be a member user or a service principal in your AAD tenant.
+
+## Step 2: Create a Microsoft Fabric Workspace
+
+Workspaces are places to collaborate with colleagues to create collections of items such as lakehouses, warehouses, and reports.
+
+1. Sign-in to your organization's PowerBI portal at https://app.powerbi.com. you would need to ask your IT admin to grant a PowerBI pro license and workspace admin rights on your account to create workspaces.
+
+2. Select workspaces from left plane and choose new workspace from bottom.
+![Ensono Stacks Fabric workspace](../images/fabric_workspaces_powerbi.png)
+
+3. Enter the name of workspace and under advanced options choose "fabric" for license mode and scroll down for step 4.
```

Comment:
Assume we'd recommend 1 workspace per environment - so perhaps advise naming these to align to the platform environments.